bimsb/packages/tainted.scm

```diff
@@ -245,7 +244,36 @@ deviation (SD) plots, coefficient of variation (CV) plots.")
                            "miRDeep2.pl"
                            "prepare_signature.pl"
                            "quantifier.pl"))
-               #t))))))
+               #t)))
+         (add-after 'wrap-perl-scripts 'test
+           (lambda* (#:key inputs outputs #:allow-other-keys)
+             (setenv "PATH" (string-append (assoc-ref outputs "out") "/bin:"
+                                           (getenv "PATH")))
+             (with-directory-excursion "tutorial_dir"
+               ;; Script writes to "~/.local/share/mirdeep2", but
+               ;; HOME=/homeless-shelter by default:
+               (setenv "HOME" ".")
+               (mkdir-p ".local/share/mirdeep2")
+               ;; Run script in bash to processs output for error detection:
+               (invoke
+                 "bash" "-c"
+                 (string-append
```

Comment:
This is not a big problem, obviously, but it would be nicer not to use `string-append` and just use a single string.  You can escape line breaks with `\`.